server/json-logger.h

```diff
@@ -41,6 +50,8 @@ class JsonLogger : vk::not_copyable {
 private:
   JsonLogger() = default;
 
+  sig_atomic_t json_logs_count{0};
```

Comment:
```suggestion
  // it's incremented from signal handler sometimes => it must be volatile sig_atomic_t to prevent data races
  volatile sig_atomic_t json_logs_count{0};
```